packages/parser/src/pipeline/classic/inferCombatEventSegments.ts

```diff
@@ -1,18 +1,35 @@
 import { Observable } from 'rxjs';
 
 import { CombatAction } from '../../actions/CombatAction';
+import { ZoneChange } from '../../actions/ZoneChange';
 import { CombatEvent, CombatUnitType, ICombatEventSegment, LogEvent } from '../../types';
 import { getUnitReaction, getUnitType, PIPELINE_FLUSH_SIGNAL } from '../../utils';
 
-const COMBAT_AUTO_TIMEOUT_SECS = 60;
+const ARENA_ZONE_NAMES = [
```

Comment:
related to the comment above - is it possible to use zone ID instead? The names would be different when played from WoW clients with different languages.

Reply:
hadn't thought about different languages, changed it to use zone ids instead